tests/integration/functional/cli/test_model.py

```diff
@@ -0,0 +1,326 @@
+        pass
+
+    def delete_model_version(
+        self,
+        name: str,
+        version: str,
+    ) -> None:
+        pass
+
+    def update_model_version(
+        self,
+        name: str,
+        version: str,
+        description: Optional[str] = None,
+        metadata: ModelRegistryModelMetadata = Field(
+            default_factory=ModelRegistryModelMetadata
+        ),
+        remove_metadata: Optional[List[str]] = None,
+        stage: Optional[ModelVersionStage] = None,
+    ) -> ModelVersion:
+        pass
+
+    def list_model_versions(
+        self,
+        name: Optional[str] = None,
+        model_source_uri: Optional[str] = None,
+        metadata: ModelRegistryModelMetadata = Field(
+            default_factory=ModelRegistryModelMetadata
+        ),
+        stage: Optional[ModelVersionStage] = None,
+        count: Optional[int] = None,
+        created_after: Optional[datetime] = None,
+        created_before: Optional[datetime] = None,
+        order_by_date: Optional[str] = None,
+        **kwargs: Any,
+    ) -> Optional[List[ModelVersion]]:
+        pass
+
+    def get_model_version(self, name: str, version: str) -> ModelVersion:
+        breakpoint()
+        pass
+
+    def load_model_version(
+        self,
+        name: str,
+        version: str,
+        **kwargs: Any,
+    ) -> Any:
+        pass
+
+    def get_model_uri_artifact_store(
+        self,
+        model_version: ModelVersion,
+    ) -> str:
+        pass
+
+
+@pytest.fixture
+def concrete_registered_models(mocker):
+    registered_models = mocker.MagicMock(spec=RegisteredModel)
+    registered_models.name = "test_model"
+    registered_models.description = "test_model_description"
+    registered_models.metadata = {"test": "test"}
+    return [registered_models]
+
+
+@pytest.fixture
+def concrete_registered_model_version(mocker, concrete_registered_models):
+    registered_model_version = mocker.MagicMock(spec=ModelVersion)
+    registered_model_version.registered_model = concrete_registered_models[0]
+    registered_model_version.version = "test_version"
+    registered_model_version.description = "test_version_description"
+    registered_model_version.model_source_uri = "test_uri"
+    registered_model_version.model_format = "test_format"
+    registered_model_version.metadata = ModelRegistryModelMetadata(test="test")
+    registered_model_version.stage = ModelVersionStage.NONE
+    registered_model_version.created_at = datetime.now()
+    registered_model_version.last_updated_at = datetime.now()
+    return [registered_model_version]
+
+
+@pytest.fixture
+def concrete_model_registry():
+    """Returns a test model registry."""
+    return ConcreteModelRegistry(
+        name="arias_registry",
+        id=uuid4(),
+        flavor="test",
+        config={},
+        type=StackComponentType.MODEL_REGISTRY,
+        user=uuid4(),
+        workspace=uuid4(),
+        created=datetime.now(),
+        updated=datetime.now(),
+    )
+
+
```

Comment:
Could you add a test that list works for model registries as well? (one that shows it's working when we have a model registry, and another that shows the alternative cases of no model registry, and also trying to use / interact with a model registry that doesn't exist?